app/src/pages/project/TracesTable.tsx

```diff
@@ -281,184 +293,204 @@ export function TracesTable(props: TracesTableProps) {
       }
       return root as SpanRowType;
     });
-
-    return tableData;
   }, [data]);
   type TableRow = (typeof tableData)[number];
 
```

Comment:
Everything below this is just indentation changes from wrapping each block in `useMemo`